pysb/importers/sbml.py

```diff
@@ -15,7 +15,10 @@
 from pysb.logging import get_logger, EXTENDED_DEBUG
 
 BIOMODELS_REGEX = re.compile(r'(BIOMD|MODEL)[0-9]{10}')
-BIOMODELS_URL = 'http://www.ebi.ac.uk/biomodels-main/download?mid={}'
+BIOMODELS_URLS = {
+    'EBI': 'http://www.ebi.ac.uk/biomodels-main/download?mid={}',
```

Comment:
Could the keys be all lowercase, `'ebi'` and `'caltech'`? That seems more pythonic to me.